node-manager/src/localstorage.rs

```diff
@@ -0,0 +1,959 @@
+use bdk::database::{BatchDatabase, BatchOperations, Database, SyncTime};
+use bdk::{KeychainKind, LocalUtxo, TransactionDetails};
+use bitcoin::consensus::deserialize;
+use bitcoin::consensus::encode::serialize;
+use bitcoin::hash_types::Txid;
+use bitcoin::{OutPoint, Script, Transaction};
+use gloo_storage::{LocalStorage, Storage};
+
+use bitcoin::hashes::hex::{FromHex, ToHex};
+use log::{error, info};
+use serde::{Deserialize, Serialize};
+use serde_json::{Map, Value};
+
+pub struct MutinyBrowserStorage {}
+
+macro_rules! log {
```

Comment:
not using this either... wonder if we could keep it somewhere top level to have for next time tho, useful for debugging (the `log` crate stuff shows up in a context where we've run `wasm_logger::init` and has some weird formatting stuff that makes it annoying outside of a real browser console)

Reply:
Yeah I can remove for now, I tried moving it to the general test mod but couldn't get it to work that way